FinderKeeper/Assets/Scripts/Inventory.cs

```diff
@@ -0,0 +1,52 @@
+﻿using System;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+
+public class Inventory : MonoBehaviour {
+
+    [SerializeField] List<ItemsDetails> items;
+    [SerializeField] Transform itemsParent;
+    public ItemsDetails collectedItems;
+    private ItemDisplay[] display;
+    public Button closeInventoryList;
+ 
+    void Update()
+    {
+        if (itemsParent != null)
+            display = itemsParent.GetComponentsInChildren<ItemDisplay>();
+        UpdateData();
+
+        Button button = closeInventoryList.GetComponent<Button>();
+        button.onClick.AddListener(CloseInventoryList);
+    }
+
+
+    public void Add(string itemName)
```

Comment:
Just nitpicking, maybe this could be called AddItem? Doubt it'd have much effect, but we may have more "add" methods across the app down the track

Reply:
Done